apps/cookbook/src/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="da">
```

Comment:
Should this be changed when the language is still english?

Reply:
Probably not 🙂